webapp/packages/core/src/dialogs/CommonDialog/DialogsPortal.tsx

```diff
@@ -69,9 +69,10 @@ function NestedDialog(props: NestedDialogType) {
   // TODO: place Dialog inside CommonDialogWrapper, so we can pass aria-label
   return styled(styles)(
     <Dialog {...dialogState} aria-label="can't be provided" tabIndex={0}>
-      <DialogBackdrop {...dialogState} onClick={handleReject} />
+      <DialogBackdrop {...dialogState} onClick={props.dialog.options?.persistent ? () => {} : handleReject} />
```

Comment:
`hideOnClickOutside` doesn't work because of onClick callback on a backdrop we manually controls dialog behavior, persistent about dialog that can't be closed by the user, this option may be used by rendering component to hide closing controls